libs/features/personalization/promo-utils.js

```diff
@@ -42,8 +42,6 @@ const getRegionalPromoManifests = (manifestNames, region, searchParams) => {
           name,
           start: GMTStringToLocalDate(start),
           end: GMTStringToLocalDate(end),
-          startUtc: start,
```

Comment:
these are values I added because I thought we needed them, but we don't.  So removing